airflow/task/task_runner/standard_task_runner.py

```diff
@@ -110,7 +110,6 @@ def terminate(self):
             return
 
         # Reap the child process - it may already be finished
```

Comment:
This comment should be removed if we’re removing the line below.

Reply:
I think it’s for the code below it where the process group was reaped?

Reply:
I *think* this comment specifically means reaping `self.process` (if it’s already finished). The block below is reaping the process *group* managed by `self.process` if it has not finished yet.

Reply:
Looks like it. Thanks